scripts/build/add-basic-constructor-to-react-component.js

```diff
@@ -0,0 +1,40 @@
+// this is to avoid the spread argument helper for react component constructors
+module.exports = function(babel) {
+  const { types: t } = babel
+
+  return {
+    name: 'ast-transform', // not required
+    visitor: {
+      Class(path) {
+        if (
+          path.node.superClass &&
+          path.node.superClass.object &&
+          path.node.superClass.object.name === 'React' &&
+          path.node.superClass.property &&
+          path.node.superClass.property.name === 'Component' &&
+          path.node.body.body.length &&
+          !path.node.body.body.some(node => {
+            return (
+              t.isClassMethod(node) &&
+              t.isIdentifier(node.key) &&
+              node.key.name === 'constructor'
+            )
+          })
+        ) {
+          path.node.body.body.unshift(
```

Comment:
lately i've become fan of `template`, IMHO it increases readability a lot - although not sure now if it will be possible to create a class method with it - we could always create dummy class with constructor only and just unpack constructor node though